safe_eth/eth/account_abstraction/user_operation.py

```diff
@@ -204,7 +204,12 @@ def gas_fees(self) -> bytes:
 
     @property
     def paymaster_and_data(self) -> bytes:
-        if not self.paymaster:
+        if (
+            not self.paymaster
+            or not self.paymaster_verification_gas_limit
+            or not self.paymaster_post_op_gas_limit
+            or not self.paymaster_data
+        ):
```

Comment:
Why are other parameters not related to this function checked here?

Reply:
All of "if" params are being used in this [return statement](https://github.com/safe-global/safe-eth-py/blob/c0609e98a31daac5872c5151dfde7cd2ea7fb6c2/safe_eth/eth/account_abstraction/user_operation.py#L214), isn't it?

Reply:
You are totally right, I was not looking at the right function 🤦🏻